src/cells.cpp

```
@@ -1,3 +1,49 @@
 //
 // Created by velocitatem on 9/27/24.
 //
+#include "cells.h"
+
+#include <map>
+
+
+ParticleLife::ParticleLife(int x, int y, string species) :
```

cppcheck

[performance] Function parameter 'species' should be passed by const reference.

(passedByValue)